# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../obexftp/obexftp-0.23-perl-vendor-install.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- obexftp-0.23/swig/perl/Makefile.am.orig	2011-05-22 00:51:21.016002362 +0200
+++ obexftp-0.23/swig/perl/Makefile.am	2011-05-22 00:52:28.740002361 +0200
@@ -9,7 +9,7 @@
 endif
 
 Makefile.perl: Makefile.PL
-	$(PERL) $< PREFIX=$(prefix) MAKEFILE=$@
+	$(PERL) $< PREFIX=$(prefix) INSTALLDIRS=vendor MAKEFILE=$@
 
 OBEXFTP.so: perl_wrap.c Makefile.perl
 	if test ! -f perl_wrap.c; then cp $(srcdir)/perl_wrap.c . ; fi
